docs/tls.md

```diff
@@ -41,3 +41,7 @@ kubectl create secret generic tls-ssl-minio --from-file=path/to/private.key --fr
 ```
 
 Once created, set the name of Secret (here it is `tls-ssl-minio`) under `spec.externalCertSecret` field. Then create the MinIOInstance. MinIO Operator will use this Secret to fetch key and certificate and mount it to relevant locations inside the MinIOInstance pods.
+
+### Using cert-manager
+
+[Certificate Manager](https://cert-manager.io) creates secrets with `tls.crt`, `tls.key`, and `ca.crt` values, which aren't compatible with MinIO by default. Certificate Manager secrets can be used by specifying the `spec.externalCertSecret.type` to be `cert-manager.io/v1alpha2`.
```

Comment:
Thanks for the PR @ioben . Do you mind adding a little more info here about how someone can leverage `cert-manager` to create certificates for MinIO server

Reply:
Sure, I've added some more information, and an example. Let me know if you'd like a different organization, or less modifications to the surrounding document